models/news.model.js

```diff
@@ -9,8 +9,19 @@ exports.selectTopics = () => {
 };
 
 exports.selectArticleById = (article_id) => {
+  const queryStr = `SELECT 
+  A.author, A.title, A.article_id, A.topic, 
+  A.created_at, A.votes, A.article_img_url, 
+  COUNT(C.article_id) AS comment_count, A.body 
+  FROM articles A 
+  JOIN comments C
```

Comment:
This should be a left join specifically so we include the articles that dont have comments as well

Reply:
Since this is a GET /api/articles/:article_id I didn't have the LEFT JOIN since the query is anyways for a specific article_id. Please let me know if this reasoning is ok.